src/Tribe/Modules/Single_Event_Template.php

```diff
@@ -135,6 +142,19 @@ public function tec_ea_single_event_template( array $event, \Tribe__Events__Aggr
 			);
 		}
 
+		// If there is a venue in the source, then change it in the template.
+		if (
+			! empty( $event['EventVenueID'] )
+			&& intval( $event['EventVenueID'] )
```

Comment:
Since negative values register as `true`...
```suggestion
			&& intval( $event['EventVenueID'] ) > 0
```